app/Http/Controllers/Admin/UserController.php

```diff
@@ -81,9 +127,15 @@ public function show($id)
      */
     public function edit($id)
     {
-        $user = User::findOrFail($id);
+        $user = User::with('roles')->findOrFail($id);
+        $roles = Role::all();
+        $rolesUser = DB::table('role_user')->where('user_id', $id)->pluck('role_id');
```

Comment:
`role_user` là bảng trung gian nhỉ, đọc thêm về Many to Many nhé. https://laravel.com/docs/5.8/eloquent-relationships#inserting-and-updating-related-models
```
$roleUser = $user->roles->pluck('id');
```